src/pandroid/app/src/main/java/com/panda3ds/pandroid/lang/Function.java

```diff
@@ -0,0 +1,5 @@
+package com.panda3ds.pandroid.lang;
```

Comment:
What is this class for? Seems unused. Same as gson, if it's for a future PR, add it in the future PR.

Reply:
Ok